psapi.go

```diff
@@ -98,3 +99,33 @@ func EnumProcesses() (pids []uint32, err error) {
 		}
 	}
 }
+
+// PerformanceInformation represents the PERFORMANCE_INFORMATION structure
+type PerformanceInformation struct {
+	CB                uint32
+	CommitTotal       uintptr
+	CommitLimit       uintptr
+	CommitPeak        uintptr
+	PhysicalTotal     uintptr
+	PhysicalAvailable uintptr
+	SystemCache       uintptr
+	KernelTotal       uintptr
+	KernelPaged       uintptr
+	KernelNonpaged    uintptr
+	PageSize          uintptr
+	HandleCount       uint32
+	ProcessCount      uint32
+	ThreadCount       uint32
```

Comment:
Could you add godoc to each field based on the API docs. e.g.

```suggestion
	ThreadCount       uint32 // The current number of threads.
```